Tests/LibWeb/Text/input/iterating-over-storage.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
Commit message: we don't normally reference the pull request itself in the commit; and as soon as it's merged, GitHub adds notes referencing to the PR anyway.

But maybe this test can simply be included in your previous commit?